ui-v2/app/components/healthcheck-list.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { get } from '@ember/object';
+
+export default Component.extend({
+  // TODO: Could potentially do this on attr change
+  actions: {
+    sortChecksByImportance: function(a, b) {
+      const statusA = get(a, 'Status');
+      const statusB = get(b, 'Status');
```

Comment:
This is where Enums are nice. This sort is essentially ordering everything Critical, then Warning, then Passing?

You could do...

```js
const precedence = {
  passing: 0,
  warning: 1,
  critical: 2,
};
// ...
sortChecksByImportance: function(a, b) {
  const statusA = precedence[get(a, 'Status')];
  const statusB = precedence[get(b, 'Status')];
  return statusB - statusA;
}
```

Reply:
Ah nice! Very helpful, thankyou.

When I come to refactor this stuff I can use this.